WordPressKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressKit"
-  s.version       = "5.0.0-beta.2"
+  s.version       = "5.0.0-beta.1"
```

Comment:
Should this be `5.0.0-beta.3` ?

Reply:
Yes!

I'll change it before merging. I just didn't it yet because I'll have to change WordPressShared too (is hardcoded to `5.0.0-beta.1`)

Reply:
Ahh.. Gotcha!